rmc/utils/constraint.py

```diff
@@ -258,7 +256,9 @@ def calculate_exp_from_mu(
     return context_ht
 
 
-def create_filtered_context_ht(overwrite: bool = True) -> None:
+def create_filtered_context_ht(
+    overwrite_temp: bool = False, overwrite: bool = True
```

Comment:
this gets called with a specific command, right? (https://github.com/broadinstitute/regional_missense_constraint/blob/true_exp/rmc/pipeline/regional_constraint.py#L48). I vote we remove the overwrite_temp and use the overwrite arg on the temp files and set `overwrite=True` on the final HT write. (I'm hoping no one will run this command just to realize the HT already exists)

Reply:
Sounds good!